example/geometry/t8_example_geometries.cxx

```diff
@@ -26,7 +26,7 @@
 #include <t8_forest/t8_forest_general.h>
 #include <t8_forest/t8_forest_io.h>
 #include <t8_geometry/t8_geometry_base.hxx>
-#include <t8_geometry/t8_geometry_implementations/t8_geometry_analytic.hxx>
```

Comment:
In anticipation for the coming changes regarding the differentiation between the c and cpp interface, it would be nice to use the cpp interface here

Reply:
I switched to the C++ interface.